doc/input-output-reference/src/overview/group-radiative-convective-units.tex

```diff
@@ -54,7 +54,7 @@ \subsubsection{Inputs}\label{inputs-038}
 
 \paragraph{Field: Rated Average Water Temperature}\label{field-rated-averagewatertemperature}
 
-This field is the rated average water temperature for the baseboard heater which is published in the manufacturer's literature in degree Celsius. It typically ranges from 65.56°C to 115.36°C in the I = B = R rating document while the lowest allowable temperature is 32.22°C. The default value is 87.78°C. If the user does not enter this field, the default value is assumed.
+This field is the rated average water temperature for the baseboard heater which is published in the manufacturer's literature in degree Celsius. It typically ranges from 65.56$^\circ$C to 115.36$^\circ$C in the I = B = R rating document while the lowest allowable temperature is 32.22$^\circ$C. The default value is 87.78$^\circ$C. If the user does not enter this field, the default value is assumed.
```

Comment:
Good change.  We should define a degree macro if we don't already have one, but that can be a different time.